### DISCUSSION TITLE
No keyboard activity when open a Elevated PowerShell session on client via MeshCentral

### DISCUSSION THREAD

**Discussion by moonwalk5** [Ylianst/MeshCentral#5430] [Q&A] [answered]
Hello all,

we have the situation, that we will open a PowerShell session as elevated user (administrator) on the client side via a MeshCentral session. The PowerShell windows is opened, but we are not able to input any key via the remote session in this box.

The user in front of the client can input any text.

This problem does not exist on the normal "cmd" prompt. This is working also from remote, if open as a elevated session.

Does anybody have the same issue and have a solution for this?

We use the latest version 1.1.13 of MeshCentral.

Thanks in advance.

With best regards.

**Comment by si458** [comment]
How are u connecting the remote device to your meshcentral?
Are you using the assistant (one click run/control) or the meshagent (install as service in background)?

**Comment by moonwalk5** [comment]
Hello si458,

I'm connected via the MeshCentral session windows, that start by click on "Connect" button.

We use on all clients the MeshCentral agent (service in background).

**Comment by moonwalk5** [comment]
Hello si458,

we have no the version of 1.1.16 of MeshCentral on Windows. But the problems further exists. 

Any idea to solve this isse?

**Reply by si458** [reply]
sadly not, i cant replicate your issue here,
whats the remote computers OS you are using?
do you just right click start icon, open powershell (admin), click yes to UAC, then type in the window?

**Comment by moonwalk5** [comment]
Hello si458,

I checked on Windows 11. I have find an hint, that using in the settings a "remote keyboard map"? Is this correct, where is this option?

With best regards,

Jörg

**Reply by si458** [reply, marked as answer]
click the 'settings' button before you click 'connect' and the setting for 'use remote keyboard map' is in there,
you can try ticking and unticking and see if any of it works?

**Comment by moonwalk5** [comment]
Hello si458,

I have checked the option with using "use remote keyboard map". If I activate this, I can now input any char in a remote elevated PowerShell.

Thanks for your support.

With best regards,

Jörg
